Bottom: f17e1ab02156d687f8752883ecdc495f911c72ea
Top:    5246ff2ab676533a9e769ce2a2e025dae9a2d5b6
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2008-10-13 00:55:11 +0100

Refresh of speedup_poly_poly_checking

---

diff --git a/src/find.c b/src/find.c
index 3fcfe9d..2ff7d64 100644
--- a/src/find.c
+++ b/src/find.c
@@ -1915,7 +1915,7 @@ LOCtoLinePourPolygon_callback (const BoxType * b, void *cl)
   PourTypePtr pour = (PourTypePtr) b;
   struct lo_info *i = (struct lo_info *) cl;
 
-  return r_search (pour->polygon_tree, (BoxType *) &i->polygon,
+  return r_search (pour->polygon_tree, (BoxType *) &i->line,
                    NULL, LOCtoLinePolygon_callback, i);
 }
